src/extlibs/hrvo/hrvo_agent.cpp

```diff
@@ -627,7 +627,7 @@ std::vector<TbotsProto::VelocityObstacle> HRVOAgent::getVelocityObstaclesAsProto
         points.emplace_back(Point(shifted_apex.x(), shifted_apex.y()));
         points.emplace_back(Point(shifted_side1.x(), shifted_side1.y()));
         points.emplace_back(Point(shifted_side2.x(), shifted_side2.y()));
-        velocity_obstacles.emplace_back(Polygon(points));
+        velocity_obstacles.emplace_back(*createVelocityObstacleProto(vo));
```

Comment:
Looks like there has been a bad merge with upstream. This function's implementation was recently updated to [this](https://github.com/UBC-Thunderbots/Software/blob/6951b2c5eb191f235ffbe30d8d5574e9da934f9b/src/extlibs/hrvo/hrvo_agent.cpp#L581).

Reply:
updated